ibc_public/utils_labels.py

```diff
@@ -0,0 +1,40 @@
+import json
+import sys
+import pandas as pd
+
+import os
+
+_package_directory = os.path.dirname(os.path.abspath(__file__))
+
+ALL_CONTRASTS = os.path.join(_package_directory, '..', 'ibc_data',
+                             'all_contrasts.tsv')
+CONTRAST_COL = 'contrast'
+
+
+def get_labels(*contrasts: str, contrast_col=CONTRAST_COL):
+    """Returns the list of labels for each passed contrast name"""
+    df = pd.read_csv(ALL_CONTRASTS, sep='\t')
+
+    for contrast in contrasts:
+        con = df[df[contrast_col] == contrast]
+
+        if len(con.index) == 0:
+            print(f"There is no contrast with the name {contrast}")
```

Comment:
I think we should raise an error here. Do you agree ?

Reply:
I agree that we should rise an error or warning. Do we want the function to be interrupted if any of the contrast names is incorrect? In that case, let's throw an error (commited). If we want to return a dictionary with all the contrasts, then maybe we can throw a warning msg for each incorrect contrast, but still return the correct ones. What do you think?

Reply:
I think that we should raise an Error. But wait I'll check more seriously  in the next days.